### ISSUE TITLE
Webhook plugin not working after updating to version 2.40

### ISSUE THREAD

**Issue by dpashev** [scm-manager/scm-webhook-plugin#55]
## Issue description
Webhook plugin not working after updating to version 2.40

### Bug Report - scm-manager.log:
2022-12-07 14:16:34.569 [ScmEventBus-1-6] [          ] ERROR com.github.legman.EventBus - ScmEventBus-1 - could not dispatch event: RepositoryHookEvent{repository=NAMESPACE/repo (1vSrlbQTk3), type=POST_RECEIVE} to handler SynchronizedEventHandler{null, public void sonia.scm.webhook.RepositoryWebHook.handleEvent(sonia.scm.repository.PostReceiveRepositoryHookEvent), true}
java.lang.reflect.InvocationTargetException: null
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at com.github.legman.InvocationContext.invoke(InvocationContext.java:108)
        at com.github.legman.InvocationContext.proceed(InvocationContext.java:101)
        at com.github.legman.micrometer.MicrometerInvocationInterceptor.invoke(MicrometerInvocationInterceptor.java:47)
        at com.github.legman.InvocationContext.proceed(InvocationContext.java:99)
        at com.github.legman.EventHandler.handleEvent(EventHandler.java:103)
        at com.github.legman.SynchronizedEventHandler.handleEvent(SynchronizedEventHandler.java:52)
        at com.github.legman.EventBus.dispatchSynchronous(EventBus.java:452)
        at com.github.legman.EventBus.lambda$dispatch$1(EventBus.java:444)
        at org.apache.shiro.subject.support.SubjectRunnable.doRun(SubjectRunnable.java:120)
        at org.apache.shiro.subject.support.SubjectRunnable.run(SubjectRunnable.java:108)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.NullPointerException: null
        at sonia.scm.webhook.RepositoryWebHook.executeWebHooks(RepositoryWebHook.java:82)
        at sonia.scm.webhook.RepositoryWebHook.handleEvent(RepositoryWebHook.java:61)
        ... 17 common frames omitted

#### Describe your SCM-Manager setup
scm-server-2.40.0-.noarch - rpm Fedora 

#### Which operating system, browser and versions do you use?
OS:  Fedora Linux 36 (Thirty Six)
Java:  /usr/bin/java --version
openjdk 11.0.17 2022-10-18
OpenJDK Runtime Environment (Red_Hat-11.0.17.0.8-2.fc36) (build 11.0.17+8)
OpenJDK 64-Bit Server VM (Red_Hat-11.0.17.0.8-2.fc36) (build 11.0.17+8, mixed mode, sharing)
Webhook plugin v2.3.2

**Comment by pfeuffer** [comment]
Hi @dpashev ,

thanks for this comprehensive report. You could do us a great favor: Could you take a look at the file `config/executedUpdates.xml` and take a look at the version for the key `sonia.scm.webhook.config.repository.xml` like this?

```
bash-5.1$ grep -A2 sonia.scm.webhook.config.repository.xml /var/lib/scm/config/executedUpdates.xml 
    <key>sonia.scm.webhook.config.repository.xml</key>
    <value>
      <latestVersion>2.1.0</latestVersion>
```

This should be `2.1.0` as show in the example. If this is not the case, somehow the update for the new plugin version has not been executed correctly.

Can you still see your configured webhooks (either globally or in the repository)?

**Comment by pfeuffer** [comment]
OK, I can reproduce the error. We will fix this asap. Sorry for the inconvenience!

**Comment by pfeuffer** [comment]
Hey @dpashev ,

the bug should be fixed now with release 2.3.3 of the webhook plugin. You should find this version in the plugin center now. We would love to hear back from you, whether this works for you now.

Sorry again for the inconvenience!

René
